### PR TITLE
Author your own plugin tutorial & CLI intialization/configuration doc

### PR DESCRIPTION
**Pull request by kaustavghosh06**

### CODE REVIEW

File: init_configure_commands_tut.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+<a href="https://aws-amplify.github.io/" target="_blank">
+    <img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="550" >
+</a>
+
+# AWS Amplify CLI initialization & configuration tutorial
+
+The AWS Amplify CLI helps you set up an AWS IAM user, set up an AWS profile on your local system and the initialize Amplify powered apps with all your defined configurations. We'll also walk through steps as to how you could share the Amplify generated backend configuration between other frontend projects and re-use the same backend infrastructure in-between multiple platforms/users.
+
+In this tutorial, we'll go walk through setting up an Amplify enabled project and go in detail as to what happens behind the scenes when doing so.
+
```
Comment (undefobj) [inline init_configure_commands_tut.md]: Can you add a TOC here with hyperlinks to the 3 areas? You just need to use a [Title](#anchor) format.

File: init_configure_commands_tut.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+# Command usage
+
+## amplify configure
+```bash
+$ amplify configure
+```
+The `amplify configure` command is a one-time setup step which helps you with the following:
```
Comment (undefobj) [inline init_configure_commands_tut.md]: I would change to this: "The amplify configure command is a one-time setup step. After you perform this on your system you only need to run amplify init going forward on new projects. If you already have the AWS CLI configured on your system you can usually skip this step. The amplify configure step helps you with the following:"

File: init_configure_commands_tut.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+```
+
+Whenever initializing a new project, the CLI would ask you to select an AWS profile and based on upon your selection all the corresponding resources are deployed to the region and IAM user tied to the selected profile.
+
+## amplify init
+```bash
+$ amplify init
+```
+
+The amplify init command is a one-time setup step for your Amplify powered cloud app which helps you with the following:
```
Comment (undefobj) [inline init_configure_commands_tut.md]: I would change to this: "The amplify init command is a one-time initialization step for your Amplify powered cloud app. You run this once for each project (JavaScript, iOS, or Android) to connect your app with an AWS backend. This setup helps you with the following:"

File: init_configure_commands_tut.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+                    }
+                }
+            }
+        ```
+        
+## amplify configure project
+```bash
+$ amplify configure project
+```
+The `amplify configure project` command should be used in the following cases:
```
Comment (undefobj) [inline init_configure_commands_tut.md]: I would change to: "The amplify configure project is an advanced command and not commonly used for getting started or individual development projects. The command should be used in the following cases:"

**Comment by kaustavghosh06** [comment]
@undefobj Thanks for the feedback. Updated the doc with your changes. I don't have permissions to merge. @mlabieniec Please merge if you think its ready to be merged.

**Pull request merged**